src/main/scala/geotrellis/sdg/PopulationNearRoadsJob.scala

```diff
@@ -54,7 +57,11 @@ class PopulationNearRoadsJob(
     countryRdd.flatMap({ country =>
       // WARN: who says these COGs exists there at all (USA does not) ?
       logger.info(s"Reading: $country ${layoutTileSource.source.name}")
-      layoutTileSource.layout.mapTransform.keysForGeometry(wsCountryBorder).map { key => (key, ())}
+      // Russian and USA rasters have large amount of NODATA regions that we want to clip to save cycles
```

Comment:
Is this commented out clip operation for USA/RUS actually worth doing? It should probably be included or the commented code just be removed.